Applications/deviceconfiguration/faraday_config.ini

```diff
@@ -12,14 +12,14 @@ BOOT_FREQUENCY_MHZ = 914.5
 BOOT_RF_POWER = 20
 
 [GPS]
-DEFAULT_LATITUDE = 012345678
```

Comment:
You completely deleted DEFAULT_LATITUDE... make sure to test code prior to pull request...